ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandServiceIT.scala

```diff
@@ -280,7 +280,11 @@ final class CommandServiceIT extends LedgerTestSuite {
         .submitAndWait(badRequest)
         .mustFail("submitting a request with a bad parameter label")
     } yield {
-      assertGrpcError(failure, Status.Code.INVALID_ARGUMENT, s"Missing record label")
+      assertGrpcError(
+        failure,
+        Status.Code.INVALID_ARGUMENT,
+        Some(Pattern.compile(s"Missing record (label|field)")),
```

Comment:
I assume the regex here is for backwards compatibility? I'm a little bit confused between this and the new exclusion you add to the compat matrix.

Reply:
The exclusion is for old ledger API test tools against new ledgers. This is to make sure we don’t need an exclusion for new ledger API test tools against old ledgers.

Reply:
👍